web/src/beta/components/Popover/context.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
```

Comment:
Unless there is a specific reason you are importing like this, can you just import what you need 🙏🏻 
In the Popover/hooks.ts file as well 🙏🏻

Reply:
Thank you. Fixed in https://github.com/reearth/reearth/pull/580/commits/eadef081850ff82f435d2e4acd4b59a5553653ad .